AISKU/tsconfig.json

```diff
@@ -4,7 +4,7 @@
     "inlineSources": true,
     "noImplicitAny": false,
     "module": "es6",
-    "target": "es5",
+    "target": "es3",
```

Comment:
NOTE: I've only changed the production code to be generated as ES3. All of the test projects can and should remain as ES5+